JASMIN/catalog-JASMIN.yaml

```diff
@@ -42,3 +42,46 @@ sources:
         max: 2015
         min: 1958
         type: int
+
+  ORCHESTRA_ZPS_CORE2NYF:
+    args:
+      urlpath: /bodc/BAS210039/CORE2NYF-ORCH0083-LIM3/{{ year }}/{{ time_freq }}/{{ grid }}/*.nc
+      xarray_kwargs:
+        compat: override
+        coords: minimal
+        data_vars: minimal
+        decode_timedelta: false
+        parallel: parallel
+    description: TODO
```

Comment:
@atb299 Could you please add a description?

Reply:
Dumb question #1, how do I add a description here? I can't find a way to edit this file. As a starter:

This is the CORE2 normal year forced simulation of the ORCHESTRA_ZPS configuration. It is a Southern Ocean domain consistent with the eORCA12, run with NEMO v3.6.

Reply:
You are maintainer now. I think you can use the GitHub editor (is there a pencil icon?) using this link: https://github.com/NOC-MSM/intake-catalogs/blob/orchestra/JASMIN/catalog-JASMIN.yaml
To get the link I did this:
1. Go to the main page `https://github.com/NOC-MSM/intake-catalogs`
2. Switch branch from master to orchestra (drop-down menu)
3. Navigate to `JASMIN/catalog-JASMIN.yaml`

From terminal:
```
git clone git@github.com:NOC-MSM/intake-catalogs.git -b orchestra
# edit JASMIN/catalog-JASMIN.yaml
git add .
git commit -m "blablabla"
git push -u origin orchestra
```